src/lib/VideoResizeMode.ts

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
there is a ResizeMode file which was already added ! this one can be removed

Reply:
Yep, agreed.
I've pushed the remove commit !
https://github.com/react-native-video/react-native-video/pull/3204/commits/32eb9d289e848f05c4470c5745594d7b99f14aaa